spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -107,6 +116,19 @@ public void init() {
 		}
 	}
 
+	@After
+	public void cleanup() {
+		try {
+			Seq<String> topics = ZkUtils.getAllTopics(kafkaTestSupport.getZkClient());
+			for (String topic: scala.collection.JavaConversions.seqAsJavaList(topics)) {
+				AdminUtils.deleteTopic(kafkaTestSupport.getZkClient(), topic);
+			}
+		}
+		catch (Exception e) {
+			log.error(e.getMessage());
```

Comment:
The tests can run with and external kafka server depending on the setting of some environment variables, check the ctor in the rule.  I don't see the need to clean up anything for embedded since everything is destroyed at the end of testing (server/zk instance are ephemeral)